sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosStoredProcedureRequestOptions.java

```diff
@@ -115,13 +116,36 @@ public CosmosStoredProcedureRequestOptions setSessionToken(String sessionToken)
         return this;
     }
 
+    /**
+     * Gets whether Javascript stored procedure logging is enabled for the current request in the Azure Cosmos DB database
+     * service or not.
+     *
+     * @return true if Javascript stored procedure logging is enabled
```

Comment:
NIT - add comment for default value

Reply:
Good point, added default value to java docs